scripts/core/superdesk-api.d.ts

```diff
@@ -1104,6 +1109,7 @@ declare module 'superdesk-api' {
         patchRaw<T extends IBaseRestApiResponse>(endpoint, id: T['_id'], etag: T['_etag'], patch: Partial<T>): Promise<T>;
         delete<T extends IBaseRestApiResponse>(endpoint, item: T): Promise<void>;
         uploadFileWithProgress<T>(endpoint: string, data: FormData, onProgress: (event: ProgressEvent) => void): Promise<T>;
+        searchTags(endpoint, params?: Dictionary<string, any>): Promise<IAutoTaggingSearchResult>;
```

Comment:
This API entry is specific to `SAMS`, so it should belong in the `SAMS API` instead:
`/scripts/extensions/sams/src/api/assets.ts`